Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -962,6 +963,16 @@ theorem Integrable.fun_smul [NormedAddCommGroup 𝕜] [SMulZeroClass 𝕜 β] [I
     {f : α → β} (hf : Integrable f μ) : Integrable (fun x ↦ c • f x) μ :=
   hf.smul c
 
+theorem Integrable.smul_enorm
+    [NormedAddCommGroup 𝕜] [SMul 𝕜 ε] [ContinuousConstSMul 𝕜 ε] [ENormSMulClass 𝕜 ε] (c : 𝕜)
+    {f : α → ε} (hf : Integrable f μ) : Integrable (c • f) μ :=
+  ⟨by fun_prop, hf.hasFiniteIntegral.smul_enorm c⟩
```

Comment:
Any chance this can become
```suggestion
  ⟨by fun_prop, by fun_prop⟩
```
?

Reply:
At the moment `HasFiniteIntegral` is not tagged with `fun_prop`: I'd rather do this in a follow-up PR.